packages/bpk-component-slider/src/BpkSlider-test.js

```diff
@@ -20,6 +20,8 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import BpkSlider from './BpkSlider';
 
+jest.mock('react-slider', () => 'react-slider');
+
```

Comment:
Is this just a good practice thing or is there a specific reason we're now mocking this out?

Reply:
It relies on a fuller browser env than is provided by `jsdom` so it breaks if not mocked

Reply:
Ok cool 👍 Thanks for clarifying